### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

File: en/SaveActions.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ Add braces encapsulating the complete field content.
...
     title = {Test with UPPERCASE word},
-    
+
 gets
 
     title = {{Test with UPPERCASE word}},
 
 However, this procedure is not recommended.
-It is better to use the "Protect Terms" functionaly.
+It is better to use the "Protect Terms" functionally.
```
Comment (matthiasgeiger) [inline en/SaveActions.md]: Should be "functionality".

File: en/TimeStamp.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The *date format* can also be customized (see below).
 The timestamp field can be edited in the **General fields** tab of the [entry editor](EntryEditor).
 If you do not see these fields, enable them at [General Fields](GeneralFields) or reset your preferences.
 
-You can manually alter the value by typing in the date and time of your choice. Also, by clicking on the calendar icon located at the right end of the field, you can select the date you want in a calendar. Additionaly, by clicking on *Today*, the current date is added. By clicking on *None*, the field is emptied. A double-click on the field sets it to today's date.
+You can manually alter the value by typing in the date and time of your choice. Also, by clicking on the calendar icon located at the right end of the field, you can select the date you want in a calendar. Additionally, by clicking on *Today*, the current date is added. By clicking on *None*, the field is emptied. A double-click on the field sets it to today's date.
```
Comment (matthiasgeiger) [inline en/TimeStamp.md]: The "today" and "none" functionality are no longer valid. Can you please remove those sentences?

File: en/SaveActions.md (hunk 1 above)
Comment (lex111) [reply en/SaveActions.md]: New to the topic, so I made such a mistake, I will change.

**Comment by lex111** [comment]
@matthiasgeiger thank you for your review, all fixed.

**Comment by matthiasgeiger** [comment]
Thank you again!

**Pull request merged**